implementations/rust/ockam/ockam_api/src/address.rs

```diff
@@ -51,6 +51,11 @@ pub fn extract_address_value(input: &str) -> Result<String, ApiError> {
     Ok(addr)
 }
 
+pub fn extract_address(input: &str) -> Result<Address, ApiError> {
+    let value = extract_address_value(input)?;
```

Comment:
I think we could remove this function and refactor `extract_address_value` so that it returns an `Address`. That would mean touching a bunch of commands. We can leave that out of this PR.